cross/borgbackup/patches/001-remove-useless-endianness-logic.patch

```diff
@@ -0,0 +1,25 @@
+Disables pseudo logic made to enforce setting of macros which are actually already defined in corresponding headers.
+
+--- setup.py
++++ setup.py
+@@ -804,20 +804,16 @@ if not on_rtd:
+                                                system_prefix=libb2_prefix, system=libb2_system,
+                                                **crypto_ext_kwargs)
+ 
+-    msgpack_endian = '__BIG_ENDIAN__' if (sys.byteorder == 'big') else '__LITTLE_ENDIAN__'
```

Comment:
It turns out that all this logic is actually useless and conflicting with the headers already provided by the cross compiling environment. Thus it should be safe to remove it.

Additionally the `sys.byteorder` information is actually stored in native resources and, as such, are not relevant when cross compiling.